src/configuration/OpenSearchDetails.test.tsx

```diff
@@ -143,10 +171,15 @@ describe('OpenSearchDetails', () => {
     const onChangeMock = jest.fn();
 
     const defaultConfig = createDefaultConfigOptions();
+    const opensearchDetectVersionValue = config.featureToggles.opensearchDetectVersion;
```

Comment:
might make sense to take this out of the describe, it's very unlikely but I guess I could see a world where we edit config.featureToggles.opesearchDetectVersion directly in a previous test up above and accidentally never reset it or something?